### ISSUE TITLE
mysqli_num_fields being passed a boolean result

### ISSUE THREAD

**Issue by johnjamesjacoby** [johnjamesjacoby/ludicrousdb#9]
I'm occasionally seeing this in my error logs:

```
PHP message: PHP Warning:  mysqli_num_fields() expects parameter 1 to be mysqli_result, boolean given in /srv/www/content/db.php on line 992
```

**Comment by johnjamesjacoby** [comment]
My hunch is that `_do_query()` is returning `false` because `dbh_type_check()` is failing.

**Comment by johnjamesjacoby** [comment]
Addressed in 887524e89aab350af3d822587c36cc53a3c114ed.

There may be a more precise way to debug what kind of connection or query is causing this, but I think it's safe to assume we'd want to suppress this error and any subsequent issues up the stack regardless.
